webapp/src/dogma/features/project/Projects.tsx

```diff
@@ -147,17 +185,22 @@ export const Projects = () => {
             <Flex gap={2}>
               <Spacer />
               <Menu>
-                <MenuButton as={Button} rightIcon={<ChevronDownIcon />}>
-                  Type
+                <MenuButton as={Button} rightIcon={<ChevronDownIcon />} colorScheme="blue">
+                  <HStack>
+                    <FaFilter />
+                    <Box>Type</Box>
+                  </HStack>
                 </MenuButton>
                 <MenuList>
                   <MenuOptionGroup
                     defaultValue={projectFilter}
                     type="radio"
-                    onChange={(type) => dispatch(setProjectFilter(type as FilterType))}
+                    onChange={(type) => dispatch(setProjectFilter(type as ProjectFilterType))}
+                    alignItems={'center'}
                   >
-                    <MenuItemOption value="all">All</MenuItemOption>
-                    <MenuItemOption value="me">Created by me</MenuItemOption>
+                    <MenuItemOption value="ALL">All</MenuItemOption>
+                    <MenuItemOption value="MEMBER">I am a member</MenuItemOption>
+                    <MenuItemOption value="CREATOR">Created by me</MenuItemOption>
```

Comment:
Do you have any easy way to test the UI in local?

Reply:
I prefer the following steps:

- Run `ShiroCentralDogmaTestServer.main()` using IntelliJ which is faster than `./gradlew runTestShiroServer`.
- If the server starts successfully, execute `npm run develop` and access `http://127.0.0.1:3000`.

If `test` is meant to run a unit test, we can run `npm run test`. I temporarily disable tests that fetch data from a mock server after upgrading dependencies. Because there were compatibility issues that I haven't solved yet.

Reply:
> Run ShiroCentralDogmaTestServer.main() using IntelliJ which is faster than ./gradlew runTestShiroServer.
If the server starts successfully, execute npm run develop and access http://127.0.0.1:3000.

That's what I was looking for. 😉